test/spec/util/ApplicationContext.test.js

```diff
@@ -19,14 +19,15 @@ describe('momo.util.ApplicationContext', function() {
 
     describe('Static methods', function() {
         describe('#loadApplicationContext', function() {
+            clazz.pathConfig.appContextUrlTpl = "../" + clazz.pathConfig.appContextUrlTpl;
             it('is defined', function() {
                 expect(clazz.loadApplicationContext).to.not.be(undefined);
             });
             it('is a function', function() {
                 expect(clazz.loadApplicationContext).to.be.a('function');
             });
             it('returns undefined if no callback function is given', function() {
-                expect(clazz.loadApplicationContext(42)).to.be(undefined);
+                expect(clazz.loadApplicationContext(662)).to.be(undefined);
```

Comment:
Hmmm, I don't think it's a good idea to use here a hard coded appId (though you also have mentioned this as TODO). 

Each time, wenn you add/remove an additional layer or component or something else and write the new context into the database, this value will be changed so this test must be continuously adapted.